fixtures/chauvet-dj/slimpar-t12-usb.json

```diff
@@ -0,0 +1,198 @@
+      "https://www.youtube.com/watch?v=Qav_XTa3I68"
+    ]
+  },
+  "physical": {
+    "dimensions": [227, 84, 252],
+    "weight": 1.8,
+    "power": 39,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "12 LEDs (tri-color RGB) 3 W (1 A)",
+      "lumens": 817
+    },
+    "lens": {
+      "degreesMinMax": [15, 15]
+    }
+  },
+  "availableChannels": {
+    "Red": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "ColorPreset",
+          "comment": "Color Macros"
+        }
+      ]
+    },
+    "Strobe/Speed": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction",
+          "comment": "When CH6 is 000-031"
+        },
+        {
+          "dmxRange": [16, 255],
+          "type": "Speed",
+          "speedStart": "slow",
+          "speedEnd": "fast",
+          "comment": "When CH6 032-223"
+        }
+      ]
+    },
+    "Mode": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 31],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [32, 63],
+          "type": "Effect",
+          "effectName": "Pulse 0-100%",
+          "comment": "CH1-3 Must be Active"
+        },
+        {
+          "dmxRange": [64, 95],
+          "type": "Effect",
+          "effectName": "Pulse 100%-0%",
+          "comment": "CH1-3 Must be Active"
+        },
+        {
+          "dmxRange": [96, 127],
+          "type": "Effect",
+          "effectName": "Pulse 100%-0%-100%",
+          "comment": "CH1-3 Must be Active"
+        },
+        {
+          "dmxRange": [128, 159],
+          "type": "Effect",
+          "effectName": "Auto Fade Transition"
+        },
+        {
+          "dmxRange": [160, 191],
+          "type": "Effect",
+          "effectName": "Auto Snap Transition (3 Colors)",
+          "comment": "Colors: RGB"
+        },
+        {
+          "dmxRange": [192, 223],
+          "type": "Effect",
+          "effectName": "Auto Snap Transition (7 Colors)",
+          "comment": "Colors: RGBCYMW"
+        },
+        {
+          "dmxRange": [224, 255],
+          "type": "Effect",
+          "effectName": "Sound Active Mode"
+        }
+      ]
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Dimmer Speed": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 51],
+          "type": "Effect",
```

Comment:
I probably would have used type "Maintenance" for setting dimmer speed, since it's not really an Effect on its own, but I'm not positive.